src/pages/playground/Decoder.tsx

```diff
@@ -0,0 +1,131 @@
+import { FSModule } from 'browserfs/dist/node/core/FS';

```

Comment:
Please fix.

Reply:
are we not supposed to use FSModule here? since we need this to refer to fileSystem and rewrite current program when receiving a new shared program link